examples/active-active/data.tf

```diff
@@ -0,0 +1,23 @@
+data "aws_secretsmanager_secret" "ca_certificate" {
+  name = var.ca_certificate_secret_name
+}
+
+data "aws_secretsmanager_secret" "ca_private_key" {
+  name = var.ca_private_key_secret_name
+}
+
+data "aws_ami" "rhel" {
+  owners = ["309956199498"] # RedHat
+
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["RHEL-7.9_HVM-*-x86_64-*-Hourly2-GP2"]
```

Comment:
This is more a comment than a request for change, but:

It feels a little weird for such a specific filter to be a non-defaulted variable. We'd want the examples to survive the test of time and this is one of those things that you forget to update until it's hosed. 

On the other hand this is a very specific targeted example, so I can see both ways here.